package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
delete this file, any npm install should be done in either frontend or backend directories